k8s/services/viz-service.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: viz-service
+  annotations:
+    cloud.google.com/backend-config: '{"default": "viz-backendconfig"}'
+spec:
+  selector:
+    app: viz
+    name: viz-pod
+  ports:
+    - protocol: TCP
+      port: 80
+      targetPort: 3007
+  type: ClusterIP
```

Comment:
Just to make sure, this limits the accessibility of this service to the cluster, right?

Reply:
This is just the type of resource we create for our services. It gives the service an IP accessible within the cluster (which is then targetted by our ingress / external load balancer)